package.json

```diff
@@ -60,7 +60,7 @@
     "eventsource-polyfill": "0.9.6",
     "express": "4.14.0",
     "express-pino-logger": "1.0.0",
-    "extract-text-webpack-plugin": "1.0.1",
+    "extract-text-webpack-plugin": "2.0.0-beta.5",
```

Comment:
according to this: https://github.com/webpack/webpack/releases
2.2.0 is only on github and this is the first 2.x version of webpack, previous one were a pre-resease so 2.0.0-rc.0 is the latest one on npm